portal-knightsconfig.json

```diff
@@ -0,0 +1,438 @@
+[
+    {
+        "DisplayName":"Server Name",
+        "Category":"Server Settings",
+        "Description":"The name that your dedicated server will display.",
+        "Keywords":"server,name",
+        "FieldName":"name",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.basicServerData.name",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"AMP Powered Portal Knights Server",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Game IP",
+        "Category":"Server Settings",
+        "Description":"The local network IP address of your dedicated server. The address 0.0.0.0 allows the host computer to listen to all local network traffic. You probably won't need to change this unless you're setting up your server to work on a specific network card or interface.",
+        "Keywords":"",
+        "FieldName":"$ApplicationIPBinding",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "Hidden":true,
+        "ParamFieldName":"$.basicServerData.ipv4",
+        "IncludeInCommandLine":false
+    },
+    {
+        "DisplayName":"Game Port",
+        "Category":"Server Settings",
+        "Description":"The port for the dedicated server.",
+        "Keywords":"",
+        "FieldName":"$GamePort",
+        "InputType":"number",
+        "IsFlagArgument":false,
+        "Hidden":true,
+        "ParamFieldName":"$.basicServerData.port",
+        "IncludeInCommandLine":false
+    },
+    {
+        "DisplayName":"Save Folder Path",
+        "Category":"Server Settings",
+        "Description":"This is the location where save data will be written.",
+        "Keywords":"Save,folder,path",
+        "FieldName":"saveFolderPath",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.basicServerData.saveFolderPath",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"./savedata"
+    },
+    {
+        "DisplayName":"Gameplay Mode",
+        "Category":"Server Settings",
+        "Description":"The game play mode for the dedicated server.",
+        "Keywords":"gameplay,Mode",
+        "FieldName":"gameplayMode",
+        "InputType":"enum",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.gameplayMode",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"Adventure",
+        "EnumValues":{
+            "Adventure": "Adventure",
+            "Creative": "Creative"
+        }
+    },
+    {
+        "DisplayName":"Universe Size",
+        "Category":"Server Settings",
+        "Description":"The universe size for the dedicated server.",
+        "Keywords":"universe,Size",
+        "FieldName":"universeSize",
+        "InputType":"enum",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.universeSize",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"Normal",
+        "EnumValues":{
+            "Small": "Normal",
```

Comment:
Why don't these match? That is, why is "Normal" not "Normal", but "Small"? Confusing?

Reply:
Lol good question. But it's right from what I tested. If you see different behavior in game then I'll change it.

Reply:
Don't understand the response. I am just saying that the UI should match what is set in the file

Reply:
It should, but it doesn't. If you make a server and open the client you'll see why it's mismatched. The devs are at fault here.

Reply:
OK. I can't run the client but happy if you are happy